pool/web/view/layout/layout_print.ml

```diff
@@ -0,0 +1,6 @@
+open Tyxml.Html
+
+let create ~document_title content =
```

Comment:
Could you combine this utils function with the one in `pool/web/utils/http_utils_print.ml`?

They are very close, only the stylesheet is different.